spec/features/ingredients/edit_ingredient_page_spec.rb

```diff
@@ -28,7 +28,7 @@
     end
 
     it 'fails to update ingredient' do
-      current_path.match(%r{/ingredients/#{ingredient.id}/edit})
+      current_path =~ %r{/ingredients/#{ingredient.id}/edit}
```

Comment:
We miss `expect` here, this thing can evaluate to `false` and test will still pass.

```ruby
expect(current_path).to match(%r{/ingredients/#{ingredient.id}/edit})
```